internal/access/access.go

```diff
@@ -128,7 +127,7 @@ func HandleAuthErr(err error, resource, operation string, roles ...string) error
 
 // Can checks if an identity has a privilege that means it can perform an action on a resource
 func Can(db *gorm.DB, identity uid.PolymorphicID, privilege, resource string) (bool, error) {
-	grants, err := data.ListGrants(db, &models.Pagination{}, data.BySubject(identity), data.ByPrivilege(privilege), data.ByResource(resource))
+	grants, err := data.ListGrants(db, nil, data.BySubject(identity), data.ByPrivilege(privilege), data.ByResource(resource))
```

Comment:
Can probably set `Limit: 1` here since the subject, privilege, resourece 3-tuple is unique